app/login/page.tsx

```diff
@@ -17,7 +17,7 @@ export default function Login({
   const router = useRouter();
   const [email, setEmail] = useState<string>('');
   const [password, setPassword] = useState<string>('');
-  const [isLoggedIn, setIsLoggedIn] = useState<boolean>(false);
+  const [isLoggedIn, setIsLoggedIn] = useState<boolean>(true);
```

Comment:
Why is this updated to true by default? That would mean that the user is always logged in.

Reply:
> Why is this updated to true by default? That would mean that the user is always logged in.

I was testing something but it will have been updated with Mai's update to the logic, so I'm sure it'll result in a merge conflict that I'll have to resolve.